model/work.py

```diff
@@ -141,13 +141,21 @@ class Work(Base):
     presentation_edition_id = Column(Integer, ForeignKey('editions.id'), index=True)
 
     # One Work may have many associated WorkCoverageRecords.
-    coverage_records = relationship("WorkCoverageRecord", backref="work")
+    coverage_records = relationship(
+        "WorkCoverageRecord", backref="work",
+        cascade="all, delete-orphan"
+    )
 
     # One Work may be associated with many CustomListEntries.
+    # However, a CustomListEntry may lose its Work without
+    # ceasing to exist.
     custom_list_entries = relationship('CustomListEntry', backref='work')
 
-    # One Work may have multiple CachedFeeds.
-    cached_feeds = relationship('CachedFeed', backref='work')
+    # One Work may have multiple CachedFeeds, and if a CachedFeed
+    # loses its Work, it ceases to exist.
+    cached_feeds = relationship(
+        'CachedFeed', backref='work', cascade="all, delete-orphan"
+    )
```

Comment:
This one is a little bit tricky because you _can_ have CachedFeeds created with work=None. But if you have a CachedFeed that _was_ associated with a Work, and then you set work=None, you're effectively deleting the feed. The CachedFeed isn't like the NYT best-seller list; it doesn't have an existence independent of the Work.